### ISSUE TITLE
6X: /bin/sh: 1: [[: not found

### ISSUE THREAD

**Issue by pivotal-ning-yu** [greenplum-db/gpdb#8688]
commit: 9e93ba8be6873fd85fc4db2a38db2981b5d294b6

```sh
$ make -C gpMgmt/bin/stream install
make: Entering directory '/home/u/src/gpdb.git/gpMgmt/bin/stream'
/bin/mkdir -p '/home/u/opt/gpdb/bin/lib'
# This install location is due to legacy reasons kept for backwards
# compatibility until it can be deprecated.
/bin/mkdir -p '/home/u/opt/gpdb/bin/stream'
/usr/bin/install -c  stream '/home/u/opt/gpdb/bin/lib/stream'
# Symlink bin/lib/stream to bin/stream to maintain backward compatibility
if [[ ! -f /home/u/opt/gpdb/bin/stream/stream && ! -L /home/u/opt/gpdb/bin/stream/stream ]]; then \
        ln -s /home/u/opt/gpdb/bin/lib/stream /home/u/opt/gpdb/bin/stream/stream; \
fi
/bin/sh: 1: [[: not found
make: Leaving directory '/home/u/src/gpdb.git/gpMgmt/bin/stream'
```

The `if` check is unnecessary at all, we could simply `ln -nfs <src> <dst>`.

However as there is also the issue https://github.com/greenplum-db/gpdb/issues/8670, the `<src>` should be a relative path instead of an absolute one.

**Comment by oliverralbertini** [comment]
If you decide to keep the check you can either make the test a POSIX-compliant one:

```bash
if ! [ -f /home/u/opt/gpdb/bin/stream/stream ] && ! [ -L /home/u/opt/gpdb/bin/stream/stream ]; then \
``` 
or ask Make to use bash instead.

**Comment by adam8157** [comment]
fixed by 8181972e4faa507af758d23ad0a1bde97a5ccdd2
